release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1785,6 +1785,10 @@ In the following tables, features are marked with the following statuses:
 
 * The DNF package manager included in {op-system-first} images cannot be used at runtime, because DNF relies on additional packages to access entitled nodes in a cluster that are under a Red Hat subscription. As a workaround, use the `rpm-ostree` command instead. (link:https://issues.redhat.com/browse/OCPBUGS-35247[*OCPBUGS-35247*])
 
+* Due to a change in storage account naming in {product-title} {product-version}, the Azure File Container Storage Interface (CSI) driver now alphabetically matches storage accounts of the Image Registry Operator. With this change, there is a known issue where the Azure File CSI driver does not work properly when the image registry is configured as private. As a result, the image registry fails to mount volumes because it does not have a worker subnet added to allowed networks for the storage account that it tries to use.
```

Comment:
```suggestion
* Due to a change in storage account naming in {product-title} {product-version}, the Azure File Container Storage Interface (CSI) driver now alphabetically matches storage account of the Image Registry Operator. With this change, there is a known issue where the Azure File CSI driver fails to mount all volumes when image registry is configured as private. The mount failures occur because the CSI driver tries to use the storage account of the Image Registry Operator which is not configured to allow connections from worker subnet.
```